### DISCUSSION TITLE
Only allow users from specific Guild

### DISCUSSION THREAD

**Discussion by SrPeterr** [JakyeRU/Larascord#18] [Ideas]
Hi!
I don't know if its possible but I want only users from my guild to be able to log-in! If they do log-in It will show what role the have :)
So I they can access some parts of the website... Is this possible?
Thanks for your time!

**Comment by ValentinNeaga** [comment]
Hello @SrPeterr,

At the moment this is not possible using Larascord. This is a great suggestion and I will try to implement it after #17 is finished. Thank you!

**Reply by SrPeterr** [reply]
Hi! Any updates on this? :(

**Reply by ValentinNeaga** [reply]
Hello. I'm sorry because this is taking so long. At the moment I don't have enough free time to implement this feature but I assure you it will get implemented.

**Comment by SrPeterr** [comment]
Any new update on this feature? 👀

**Reply by ValentinNeaga** [reply]
Hello,

I've opened PR #31 which will add this feature.

**Reply by ValentinNeaga** [reply]
@SrPeterr This feature was released in [v3.2.0](https://github.com/JakyeRU/Larascord/releases/tag/v3.2.0).

**Reply by SrPeterr** [reply]
> @SrPeterr This feature was released in [v3.2.0](https://github.com/JakyeRU/Larascord/releases/tag/v3.2.0).

Amazing, thank you!
As a suggestion for future updates would be great to get the roles from that user in the guild :)

**Reply by ValentinNeaga** [reply]
Thank you for the great suggestion. At the moment there are no plans for implementing this feature but could get implemented in the future.
